README.md

```diff
@@ -1 +1,106 @@
-# gha-ci
\ No newline at end of file
+# GitHub Actions - CI
+
+CI for Silverstripe modules
+
+Uses feature detection based on files in the root folder such as phpunit.xml.dist to build dynamic matrix of jobs to run
+
+It's highly recommended that you use a tagged version (e.g. v1) to ensure stability of your builds. If you have a relatively simple build that you have no intention of ever making more complex e.g. only phpunit tests using phpunit.xml.dist, then this is probably all you need for long term use.
+
+Note: Unlike other silverstripe/gha-* repositories, this one is a [reusable workflow](https://docs.github.com/en/actions/using-workflows/reusing-workflows) rather than an [action](https://docs.github.com/en/actions/creating-actions/creating-a-composite-action). A reusable workflow is required to create a `matrix`.
```

Comment:
Yeah it's more of a dev note, though it's kind of an important one since someone will at some point wonder why this isn't an action.  I'll probably wonder myself in 3 months :-) The repo does look weird because the yml lives in `.github/workflows/ci.yml` rather than `action.yml` like everything else.

Reply:
Yup, I agree that the comment as a whole is useful. It's just the part that I've recommended removing that seems unnecessary and arguably confusing. I can see someone thinking "Oh does that mean I can just throw my own matrix in there?"
Though I guess they _kind of_ can, if they turn dynamic matrix off and pass in their own extra jobs. So I guess it's fine.